### DISCUSSION TITLE
A project with metro config.js won't start

### DISCUSSION THREAD

**Discussion by Revaycolizer** [danstepanov/create-expo-stack#392] [Q&A]
I initially initialized a new project with create expo stack cli but when I ran the development server it won't finish rendering it stacks at 45% or 54%

**Comment by dannyhw** [comment]
@Revaycolizer please can you share your cesconfig.json so we can understand what options your picked. Also any more details about the error.

**Reply by Revaycolizer** [reply]
```
{
  "cesVersion": "2.11.17",
  "projectName": "test",
  "packages": [
    {
      "name": "expo-router",
      "type": "navigation",
      "options": {
        "type": "drawer + tabs"
      }
    },
    {
      "name": "nativewind",
      "type": "styling"
    }
  ],
  "flags": {
    "noGit": false,
    "noInstall": false,
    "overwrite": false,
    "importAlias": true,
    "packageManager": "bun",
    "eas": false
  },
  "packageManager": {
    "type": "bun",
    "version": "1.1.8"
  },
  "os": {
    "type": "Windows_NT",
    "platform": "win32",
    "arch": "x64",
    "kernelVersion": "10.0.22621"
  }
}
```

**Reply by Revaycolizer** [reply]
![Screenshot 2024-08-12 121857.png](https://github.com/user-attachments/assets/396efdd9-37d3-4f49-8d6e-0936fa4508a3)

It stacks here at this percentage and it doesn't reach to 100%

**Reply by dannyhw** [reply]
Hmm I'm not sure, for me its working as normal

**Reply by dannyhw** [reply]
can you try clearing the cache just in case?

**Reply by Revaycolizer** [reply]
I tried with bunx expo start -c but it stacks at 27%

**Reply by dannyhw** [reply]
🤔 I will try on windows machine a bit later and see if im able to reproduce

**Reply by Revaycolizer** [reply]
Thanks a lot, seems in windows there is a problem as I tried in Ubuntu and it works properly
